src/app/layout.tsx

```diff
@@ -10,12 +10,12 @@ const libreFranklin = Libre_Franklin({
 
 export const metadata: Metadata = {
   title: "Kernel Community",
-  description: "Building a better web together, block by block. Block 10 Applications now open.",
+  description: "Cultivating a greater web, together. Applications open for KBX.",
   openGraph: {
     title: 'Kernel Community',
-    description: 'Building a better web together, block by block. Block 10 Applications now open.',
+    description: 'Cultivating a greater web, together. Applications open for KBX.',
```

Comment:
Note that I followed the same single quote format as was originally in the repo, instead of doing double quotes.